opendbc/car/__init__.py

```diff
@@ -100,117 +92,6 @@ class Bus(StrEnum):
-  return float(val)
-
-
-def apply_meas_steer_torque_limits(apply_torque, apply_torque_last, motor_torque, LIMITS):
-  return int(round(apply_dist_to_meas_limits(apply_torque, apply_torque_last, motor_torque,
-                                             LIMITS.STEER_DELTA_UP, LIMITS.STEER_DELTA_DOWN,
-                                             LIMITS.STEER_ERROR_MAX, LIMITS.STEER_MAX)))
-
-
-def apply_std_steer_angle_limits(apply_angle: float, apply_angle_last: float, v_ego: float, steering_angle: float,
-                                 lat_active: bool, limits: AngleSteeringLimits) -> float:
-  # pick angle rate limits based on wind up/down
-  steer_up = apply_angle_last * apply_angle >= 0. and abs(apply_angle) > abs(apply_angle_last)
-  rate_limits = limits.ANGLE_RATE_LIMIT_UP if steer_up else limits.ANGLE_RATE_LIMIT_DOWN
-
-  angle_rate_lim = np.interp(v_ego, rate_limits[0], rate_limits[1])
-  new_apply_angle = np.clip(apply_angle, apply_angle_last - angle_rate_lim, apply_angle_last + angle_rate_lim)
-
-  # angle is current steering wheel angle when inactive on all angle cars
-  if not lat_active:
-    new_apply_angle = steering_angle
-
-  return float(np.clip(new_apply_angle, -limits.STEER_ANGLE_MAX, limits.STEER_ANGLE_MAX))
-
-
-def common_fault_avoidance(fault_condition: bool, request: bool, above_limit_frames: int,
-                           max_above_limit_frames: int, max_mismatching_frames: int = 1):
-  """
-  Several cars have the ability to work around their EPS limits by cutting the
-  request bit of their LKAS message after a certain number of frames above the limit.
-  """
-
-  # Count up to max_above_limit_frames, at which point we need to cut the request for above_limit_frames to avoid a fault
-  if request and fault_condition:
-    above_limit_frames += 1
-  else:
-    above_limit_frames = 0
-
-  # Once we cut the request bit, count additionally to max_mismatching_frames before setting the request bit high again.
-  # Some brands do not respect our workaround without multiple messages on the bus, for example
-  if above_limit_frames > max_above_limit_frames:
-    request = False
-
-  if above_limit_frames >= max_above_limit_frames + max_mismatching_frames:
-    above_limit_frames = 0
-
-  return above_limit_frames, request
-
-
-def apply_center_deadzone(error, deadzone):
-  if (error > - deadzone) and (error < deadzone):
-    error = 0.
-  return error
-
-
-def rate_limit(new_value, last_value, dw_step, up_step):
-  return float(np.clip(new_value, last_value + dw_step, last_value + up_step))
-
-
-def get_friction(lateral_accel_error: float, lateral_accel_deadzone: float, friction_threshold: float,
-                 torque_params: structs.CarParams.LateralTorqueTuning) -> float:
-  friction_interp = np.interp(
-    apply_center_deadzone(lateral_accel_error, lateral_accel_deadzone),
-    [-friction_threshold, friction_threshold],
-    [-torque_params.friction, torque_params.friction]
-  )
-  return float(friction_interp)
-
-
```

Comment:
it's best to make unrelated changes in separate PRs. this should be not changed though

Reply:
Yeah, I was thinking this made more sense being on `lateral.py` considering it's a function used explicitly for lateral control, but at the same time I was hesitant on whether it was used because I was looking for usages in the wrong place.

Are we okay with it being on `lateral.py` ? I will update the reference in OP too if so

Reply:
if we are good, there's this then:
* https://github.com/commaai/openpilot/pull/35856